frontend/lib/hp-action.tsx

```diff
@@ -180,7 +180,8 @@ const HPActionConfirmation = withAppContext((props: AppContextType) => {
   const href = props.session.latestHpActionPdfUrl;
 
   return (
-    <Page title="Your HP Action packet is ready!" withHeading className="content">
+    <Page title="Your HP Action packet is ready!" className="content">
+      <h1 className="title">Your HP Action packet is ready!</h1>
```

Comment:
Oh, I am assuming you did this to make the heading bigger?  If you look at the source for the `<Page>` component you'll see that `withHeading` can be boolean, but it can also be the string `"big"` or `"small"`, and if you set it to `"big"`, it should do what you want, without you needing to repeat the same title twice.  (I'm concerned about more than just laziness, which is that we might alter it in one place but forget to alter it in the other--and it's easy to happen since the page's `<title>` is only easily visible to screen readers!)

Reply:
Also wait, I am confused, this is in `hp-action.tsx` but the title of this PR is about the rental history flow.  Did you mean to make this change?

Reply:
Oops scrach the last comment, I hadn't read [your earlier comment](https://github.com/JustFixNYC/tenants2/pull/873#issuecomment-534726694) until just now.

Reply:
nice catch— just fixed it!